AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -175,6 +176,15 @@ - (void)setBackgroundColor:(UIColor *)backgroundColor
   _placeholderTextKitComponents.textView.backgroundColor = backgroundColor;
 }
 
+- (void)setTextContainerInset:(UIEdgeInsets)textContainerInset
+{
+  ASDN::MutexLocker l(_textKitLock);
+
+  _textContainerInset = textContainerInset;
+  _textKitComponents.textView.textContainerInset = textContainerInset;
```

Comment:
In the future I'd still prefer we be more explicit about this — a comment is fine if it is logically certain that an if (nodeLoaded) check is not needed.  As a future reader / author in this code, though, it isn't immediately obvious that these latter two lines are intended to be no-ops and should not trigger the views to be created.